lib/common.bash

```diff
@@ -28,9 +31,39 @@ KATA_TESTS_CACHEDIR="${KATA_TESTS_CACHEDIR:-${KATA_TESTS_BASEDIR}/cache}"
 KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
 experimental_qemu="${experimental_qemu:-false}"
 
+# Display a message to stderr, a dump of lots of useful debug
+# information (including a full stacktrace) and exit 1.
 die() {
 	local msg="$*"
-	echo -e "[$(basename $0):${BASH_LINENO[0]}] ERROR: $msg" >&2
+
+	echo >&2 "ERROR: $msg"
+
+	[ -z "${CI:-}" ] && exit 1
```

Comment:
If you run something manually, don't you expect to see the same kind of stack trace? What about having a dedicated variable for stack traces, which each developer can set the way they want, and that we set the right way for CI builds?

Also, this means that you print less information than before in that case, which I think is undesirable.

Reply:
Yep - I did think about this. I've now changed the function so that verbose mode is enabled if:

- Running under the CI (`CI=true`)
- Manually requested (`KATA_TEST_VERBOSE=true`)